Release 2.6.1 (#440)
nautobot/nautobot-app-ssot: docs/admin/release_notes/version_2.6.md

- [#418](https://github.com/nautobot/nautobot-app-ssot/issues/418) - Unpins multiple dependencies.
- [#421](https://github.com/nautobot/nautobot-app-ssot/issues/421) - Opened prometheus-client dependency range and removed direct drf-spectacular dependency.

## [v2.6.1 (2024-04-29)](https://github.com/nautobot/nautobot-app-ssot/releases/tag/v2.6.1)

### Added

- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Added additional unit tests for Device42 integration.

### Changed

> - [#437](https://github.com/nautobot/nautobot-app-ssot/issues/437) - Changed the Infoblox `utils.client` to make API calls using `requests.Session` instead of `requests.request`.

Reviewer: ```suggestion
- [#437](https://github.com/nautobot/nautobot-app-ssot/issues/437) - Improved performance of the Infoblox client by using `requests.Session` for API calls instead of `requests.request`.
```

### Fixed

> - [#431](https://github.com/nautobot/nautobot-app-ssot/issues/431) - Updated note on nautobot_ssot/integrations/ipfabric/diffsync/adapter_ipfabric.py IPFabricDiffSync from Nautobot to IPFabric.

Reviewer: This docstring change should be under housekeeping.

- [#435](https://github.com/nautobot/nautobot-app-ssot/issues/435) - Fixed handling of DLM App installed but not enabled throwing RuntimeError.
- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed IPAddress attribute to be ip_version.

> - [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed IPAddress Status to Active if available, else Reserved.

Reviewer: Is this fixing an error when the "Active" status isn't available? If so, it would be more clear to say "Fixed an error when creating IPAddress objects when the `Active` Status is not present."

Author: I believe that's referring to [this](https://github.com/nautobot/nautobot-app-ssot/pull/436/commits/e06b5fb4757761d9aa03e476f1d56b4d5f19d8d9) commit.

Reviewer: Oh, it looks like the IP Address was incorrectly being set to reserved if it was "available"? If so:

```suggestion
- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed IPAddress Status incorrectly being set to Reserved when the status in Device42 showed as "available".
```

Author: Correct!

> - [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed IPAddress assignment to Interface and setting primary.
> - [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed variable used when assigning IPAddress to Interface.

Reviewer: Perhaps?
```suggestion
- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed multiple bugs when assigning IPAddresses to Interfaces.
```

### Added

- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Added additional unit tests for Device42 integration.

### Changed

- [#437](https://github.com/nautobot/nautobot-app-ssot/issues/437) - Improved performance of the Infoblox client by using `requests.Session` for API calls instead of `requests.request`.

### Fixed

- [#435](https://github.com/nautobot/nautobot-app-ssot/issues/435) - Fixed handling of DLM App installed but not enabled throwing RuntimeError.
- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed IPAddress attribute to be ip_version.
- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed IPAddress Status to Active if available, else Reserved.
- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed multiple bugs when assigning IPAddresses to Interfaces.
- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed check for Building definiton when creating a VLAN.
- [#436](https://github.com/nautobot/nautobot-app-ssot/issues/436) - Fixed VLAN to use location instead of location_id in create().

> ### Houstkeeping

Reviewer: ```suggestion
### Housekeeping
```

Author: Oops! Good catch!
